cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -220,6 +220,9 @@ func Run(s *options.CMServer) error {
 	panic("unreachable")
 }
 
+// StartControllers is start controller (replication controller,node controller,resourceQuota controller,
+// namespace controller,serviceAccount controller,token controller,server controller,
+// endpoint controller) in Controller manager
```

Comment:
I suggest "StartControllers starts all the controllers."

I do not recommend putting the list in the comment, since it will likely become quickly out of date.

Additionally--sorry to be nitpicky--but if you do want to have the list, there need to be spaces after the commas :)